libassuan will fail pkgconf test pipeline

If the pkgconf test pipeline is added to libassuan it will fail in the following way:

```
2024/11/12 23:11:27 INFO running step "test/pkgconf"
2024/11/12 23:11:27 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:11:27 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:11:27 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:11:27 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:11:27 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + basename /home/build/melange-out/libassuan-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + dev_pkg=libassuan-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + apk info -L libassuan-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + grep -q ^Name: usr/lib/pkgconfig/libassuan.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + basename usr/lib/pkgconfig/libassuan.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + lib_name=libassuan uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + echo usr/lib/pkgconfig/libassuan.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + grep -q '^usr/lib/pkgconfig/libassuan.pc$\|^usr/share/pkgconfig/libassuan.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + pkgconf --exists libassuan uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + grep -q ^Version: usr/lib/pkgconfig/libassuan.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + pkgconf --modversion libassuan uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 INFO 3.0.1 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + grep -q ^Libs: usr/lib/pkgconfig/libassuan.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + pkgconf --libs libassuan uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 INFO -lassuan uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + grep -q ^Cflags: usr/lib/pkgconfig/libassuan.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN + pkgconf --cflags libassuan uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN Package gpg-error was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN Perhaps you should add the directory containing `gpg-error.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 WARN Package 'gpg-error', required by 'libassuan', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:11:27 INFO pod 0ef1f7c7ab4da048783015875e5069a4f97035001440db0f85800b52350742a4 terminated
2024/11/12 23:11:27 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:11:27 INFO   workspace dir: /home/user/tmp/melange-workspace-2385513761
2024/11/12 23:11:27 INFO   guest dir: /home/user/tmp/melange-guest-2041937874
2024/11/12 23:11:27 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/libassuan] Error 1
```

--- a/libassuan.yaml
+++ b/libassuan.yaml
@@ -1,7 +1,7 @@
 package:
   name: libassuan
   version: 3.0.1
-  epoch: 0
+  epoch: 1
   description: IPC library used by some GnuPG related software
   copyright:
     - license: LGPL-2.1-or-later
@@ -38,7 +38,11 @@ subpackages:
     dependencies:
       runtime:
         - libassuan
+        - libgpg-error-dev
     description: libassuan dev
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
   - name: libassuan-doc
     pipeline:

